src/dotnet/commands/dotnet-new/CSharp_nunittest/project.json.template

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable"
+  },
+  "dependencies": {
+    "System.Runtime.Serialization.Primitives": "4.1.1",
+    "NUnit": "3.4.1",
+    "dotnet-test-nunit": "3.4.0-beta-1"
+  },
+  "testRunner": "nunit",
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0"
+        }
+      },
+      "imports": [
+        "dotnet5.4",
+        "portable-net451+win8"
+      ]
```

Comment:
I have the imports for portable and AFAIK, the import for dotnet is deprecated and not needed.